### PR TITLE
feat(test) : Adding test_uzfs.sh to openebs/libcstor from openebs/cstor

### PR DESCRIPTION
**Pull request by mynktl**
Signed-off-by: mayank <mayank.patel@mayadata.io>

### CODE REVIEW

File: tests/cbtest/script/test_uzfs.sh [hunk 1]
```diff
@@ -0,0 +1,1361 @@
+#!/bin/bash
+#
+#  (C) Copyright 2017 CloudByte, Inc.
```
Comment (vishnuitta) [inline tests/cbtest/script/test_uzfs.sh]: we need to change the license header

Comment (mynktl) [reply tests/cbtest/script/test_uzfs.sh]: ok. Should I add openebs license header?

File: .travis.yml [hunk 2]
```diff
@@ -64,7 +64,7 @@ install:
 script:
     export FIO_SRCDIR=$PWD/../fio;
     sudo bash ./print_debug_info.sh &
-    sudo bash ../libcstor/tests/cbtest/script/test_uzfs.sh -T all || travis_terminate 1;
+    sudo bash ../libcstor/tests/cstor/script/test_uzfs.sh -T all || travis_terminate 1;
```
Comment (vishnuitta) [inline .travis.yml:67]: :+1:

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**